staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -658,17 +656,17 @@ func (meal *cfgMeal) finishQueueSetReconfigsLocked() {
 // object is malformed in a way that is a problem for this package.
 func queueSetCompleterForPL(qsf fq.QueueSetFactory, queues fq.QueueSet, pl *flowcontrol.PriorityLevelConfiguration, requestWaitLimit time.Duration, intPair metrics.TimedObserverPair) (fq.QueueSetCompleter, error) {
 	if (pl.Spec.Type == flowcontrol.PriorityLevelEnablementExempt) != (pl.Spec.Limited == nil) {
-		return nil, errors.New("broken union structure at the top")
+		return nil, fmt.Errorf("broken union structure at the top")
```

Comment:
I thought `errors.New` was recommended when there's no arguments? (but referring to the std lib `"errors"`, not this other package)

Reply:
Occurs twice below, too. Otherwise the change looks fine.

Reply:
> I thought `errors.New` was recommended when there's no arguments? (but referring to the std lib `"errors"`, not this other package)

I changed to the lib `errors` and changed `errors.New` back, thanks for reminding